### DISCUSSION TITLE
how to pass argument/variable when using cron

### DISCUSSION THREAD

**Discussion by cometta** [apalis-dev/apalis#387] [Q&A] [answered]
referring to this example

https://github.com/geofmureithi/apalis/blob/353d6f8450e862daa3553cb953a73a15bdd9cea4/examples/async-std-runtime/src/main.rs#L48

how to pass extra data/variables to the cron job's function `send_reminder`. Can you show example?

**Comment by geofmureithi** [comment, marked as answer]
Use the .data method
https://github.com/geofmureithi/apalis/blob/master/examples/basics/src/main.rs#L104
